lib/coins/litecoin/address.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:cupcake/coins/abstract/address.dart';
+
+class LitecoinAddress2 implements Address {
+  LitecoinAddress2(this.address);
```

Comment:
I assume this is because `LitecoinAddress` is already in some other dependency, but I would prefer if we rename it to something like `CupCakeLitecoinAddress` or so.
Also what's the other dependency that's using the name? bitcoin_base?